src/unicode.cpp

```diff
@@ -44,7 +46,27 @@ constexpr bool to_lower_ascii(char* input, size_t length) noexcept {
   }
   return non_ascii == 0;
 }
-#if ADA_NEON
+#endif
+#if ADA_SVE
+ada_really_inline bool has_tabs_or_newline(
+    std::string_view user_input) noexcept {
+  const svuint8_t mask1 = svdup_n_u8('\r');
+  const svuint8_t mask2 = svdup_n_u8('\n');
+  const svuint8_t mask3 = svdup_n_u8('\t');
+  svbool_t running = svdup_n_b8(false);
+  const size_t lanes = svcntb();
```

Comment:
depending on the lane, the remaining items needs to be checked as well. for example, user_input.size() % lanes

Reply:
No, Arm SVE has masking instructions, so we doesn't need to consider about _remaining_ elements.